src/test/java/com/google/api/codegen/testdata/nodejs_library.baseline

```diff
@@ -0,0 +1,862 @@
+ *
+ * @param {String} name
+ *   The name of the shelf we're adding books to.
+ * @param {String} otherShelfName
+ *   The name of the shelf we're removing books from and deleting.
+ * @param {?gax.CallOptions} options
+ *   Overrides the default settings for this call, e.g, timeout,
+ *   retries, etc.
+ * @param {?APICallback<google.example.library.v1.Shelf>} callback
+ *   The function which will be called with the result of the API call.
+ * @throws an error if the RPC is aborted.
+ */
+LibraryServiceApi.prototype.mergeShelves = function mergeShelves(
+    name,
+    otherShelfName,
+    options,
+    callback) {
+  var args = arguejs({
+    'name': String,
+    'otherShelfName': String,
+    'options': [gax.CallOptions],
+    'callback': [Function]
+  }, arguments);
+  var req = {
+    'name': args.name,
+    'other_shelf_name': args.otherShelfName
+  };
+  this.apiCallables.mergeShelves.then(function(apiCall) {
+    apiCall(req, args.callback, options, this.headers, {});
+  }.bind(this), function(err) {
+    if (args.callback) {
+      args.callback(err);
+    }
+  });
+};
+
+/**
+ * Creates a book.
+ *
+ * @param {String} name
+ *   The name of the shelf in which the book is created.
+ * @param {google.example.library.v1.Book} book
+ *   The book to create.
+ * @param {?gax.CallOptions} options
+ *   Overrides the default settings for this call, e.g, timeout,
+ *   retries, etc.
+ * @param {?APICallback<google.example.library.v1.Book>} callback
+ *   The function which will be called with the result of the API call.
+ * @throws an error if the RPC is aborted.
+ */
+LibraryServiceApi.prototype.createBook = function createBook(
+    name,
+    book,
+    options,
+    callback) {
+  var args = arguejs({
+    'name': String,
+    'book': Object,
+    'options': [gax.CallOptions],
+    'callback': [Function]
+  }, arguments);
+  var req = {
+    'name': args.name,
+    'book': args.book
+  };
+  this.apiCallables.createBook.then(function(apiCall) {
+    apiCall(req, args.callback, options, this.headers, {});
+  }.bind(this), function(err) {
+    if (args.callback) {
+      args.callback(err);
+    }
+  });
+};
+
+/**
+ * Creates a series of books.
+ *
+ * @param {google.example.library.v1.Shelf} shelf
+ *   The shelf in which the series is created.
+ * @param {google.example.library.v1.Book[]} books
+ *   The books to publish in the series.
+ * @param {?Object} otherArgs
+ * @param {number} otherArgs.edition
+ *   The edition of the series
+ * @param {?gax.CallOptions} options
+ *   Overrides the default settings for this call, e.g, timeout,
+ *   retries, etc.
+ * @param {?APICallback<google.example.library.v1.PublishSeriesResponse>} callback
+ *   The function which will be called with the result of the API call.
+ * @throws an error if the RPC is aborted.
+ */
+LibraryServiceApi.prototype.publishSeries = function publishSeries(
+    shelf,
+    books,
+    otherArgs,
+    options,
+    callback) {
+  var args = arguejs({
```

Comment:
From https://docs.omniref.com/js/npm/arguejs/0.1.2#label-Getting+started:

> you will not use conventional parameters definition as you used before. Actually, all your methods should be defined without them.

Is this syntax correct?